docs/projects/variables/google-cloud-account-variables.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Google Cloud account variables
+description: Create a Google Cloud account variable to use it in Google Cloud deployment steps
+position: 65
+---
+
+:::hint
+Google Cloud Accounts were added in Octopus **2021.2**.
+:::
+
+[Google Cloud accounts](/docs/infrastructure/accounts/google-cloud/index.md) are included in a project through a project [variable](/docs/projects/variables/index.md) of the type **Google Cloud Account**. Before you create a **Google Cloud account Variable**, you need to [create a Google Cloud account](/docs/infrastructure/accounts/google-cloud/index.md) in Octopus:
+
+![Google Cloud account variable](images/google-cloud-account-variable.png "width=500")
+
+The **Add Variable** window is then displayed and lists all the Google Cloud accounts.
+
+Select the Google Cloud account you want to access from the project to assign it to the variable:
+
+![Google Cloud account variable selection](images/google-cloud-account-variable-selection.png "width=500")
+
+## Google Cloud account variable properties
+
+The Google Cloud account variable also exposes the following properties that you can reference in a PowerShell script:
+
+| Name and description | Example |
+| -------------------- | ------------------------|
+| **`JsonKey`** <br/> The JSON Key for the Google Cloud account| |
```

Comment:
Given this table does not have examples (the actual example is below), do we need the "Example" column?

Reply:
I'll remove this column. Probably have to do the same for AWS and Azure accounts